R/available.R

```diff
@@ -65,7 +65,11 @@ create <- function(name, ...) {
 #' @export
 suggest <- function(path = ".", title = NULL) {
   if (is.null(title)) {
-    title <- unname(desc::desc(path)$get("Title"))
```

Comment:
oh, i just meant that `path` is the path to a **package**, so the error would be more informative with something like `"Please specify a path to an R package that contains a DESCRIPTION file"` - but more concise.